dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -222,5 +180,11 @@
     = text_field_tag 'level[contained_level_names][]', level_name, placeholder: 'name of contained level'
   #plusanswer
     %i.fa.fa-plus-circle
-  :javascript
-    $("#plusanswer").on("click", function () { $("#plusanswer").prev().clone().insertBefore("#plusanswer") });
+
+%script{src: minifiable_asset_path('js/levels/editors/_blockly.js'),
+        data: {pageOptions: {skin_id: @level.skin,
+                             isK1: @level.try(:isK1),
+                             app: @level.game.app,
+                             solution_blocks: @level.respond_to?(:solution_blocks),
+                             uses_droplet: @level.game.uses_droplet?,
+                             input_output_table: @level.respond_to?(:input_output_table)}.to_json}}
```

Comment:
so I've tried the former before and haml really does not like multiline tags like that. Each line must end with a comma in order for the next line to get parsed correctly 👎 

> Attribute hashes can also be stretched out over multiple lines to accommodate many attributes. However, newlines may only be placed immediately after commas.

See [docs](http://haml.info/docs/yardoc/file.REFERENCE.html#attributes_)

But I didn't know about the `:ruby` declaration. that looks like the way to go!